src/cli/plugin.rs

```diff
@@ -0,0 +1,116 @@
+use crate::serve_session::ServeSession;
+
+use memofs::InMemoryFs;
+use memofs::{Vfs, VfsSnapshot};
+use roblox_install::RobloxStudio;
+use snafu::{ResultExt, Snafu};
+use std::{
+    fs::{self, File},
+    io::{self, BufWriter},
+};
+use structopt::StructOpt;
+
+static PLUGIN_FILE_NAME: &str = concat!("rojo-", env!("CARGO_PKG_VERSION"), ".rbxmx");
```

Comment:
When a user upgrades their Rojo version, I think this will result in duplicate copies of the plugin getting loaded.

I'm not sure what a good solution to this is. I think it might be a good idea to install the plugin to the same path every time for now, and give it a name that's specific to the version the binary installs, like `RojoManagedPlugin.rbxm`.

Reply:
Yep I hesitated a lot with this too, I chose that in case some people would have different versions of rojo installed... like if they were trying an alpha release but wanted to keep their other rojo at the same time. I don't know if it's really a concern though.

It's probably more practical for the majority of useres to have a single name like you suggested.